candle/standard/opentheory/prettyScript.sml

```diff
@@ -0,0 +1,491 @@
+open preamble holSyntaxTheory mlstringTheory
+
+val _ = new_theory "pretty";
```

Comment:
This theory should not be here. I think it should be in the basis library.

Reply:
At least, the part that's not specific to HOL terms. (Although @myreen also wants the Candle kernel to be in the basis library..)